alicloud/service_alicloud_ecs.go

```diff
@@ -806,3 +806,27 @@ func (s *EcsService) DescribeSnapshotById(snapshotId string) (*ecs.Snapshot, err
 	}
 	return &resp.Snapshots.Snapshot[0], nil
 }
+
+func (s *EcsService) DescribeLaunchTemplate(id string) (set ecs.LaunchTemplateVersionSet, err error) {
+	req := ecs.CreateDescribeLaunchTemplateVersionsRequest()
+	req.LaunchTemplateId = id
+
+	raw, err := s.client.WithEcsClient(func(ecsClient *ecs.Client) (interface{}, error) {
+		return ecsClient.DescribeLaunchTemplateVersions(req)
+	})
+	if err != nil {
+		if IsExceptedError(err, "InvalidLaunchTemplate.NotFound") {
+			err = WrapErrorf(err, NotFoundMsg, ProviderERROR)
```

Comment:
This error is a api error, so the line 819 should be `err = WrapErrorf(err, NotFoundMsg, AlibabaCloudGoSdkERROR)`